src/views/settings/Storage/ObjectStorage.vue

```diff
@@ -75,6 +75,12 @@ export default {
               canDelete: function({ row }) {
                 return (row.name !== 'default' && row.name !== 'null' && vm.$hasPerm('terminal.delete_replaystorage'))
               },
+              onUpdate: ({ row }) => {
+                vm.$route.params.id = row.id
+                vm.$route.query.type = row.type.value
+
+                this.$refs.ListTable.onUpdate({ row })
+              },
               hasClone: false,
               extraActions: [
                 {
```

Comment:
The code is mostly clean with only minor syntax errors. No major issues were found and no optimization suggestions can be made from current information.